CHANGES.rst

```diff
@@ -1,3 +1,11 @@
+Developing 2024-03-26
+
+-   Fix the type check failure caused by inconsistency between
+    `SQLAlchemy().relationship` and `sqlalchemy.orm.relationship`. :issue:`1318`
+-   Add `allowlist_externals` property to `tox.ini` to prevent the failure that `mypy`
```

Comment:
This is a contributors-only change, right? I don't think we'd typically put that in the CHANGES. @davidism can correct me if I'm wrong.

Reply:
+1, from an end user point of view i would not care about such things...

Reply:
Thank you for letting me know! I will remove this part after I rebase this PR.